src/sidebar/components/group-list.js

```diff
@@ -26,6 +40,18 @@ function GroupListController($window, analytics, groups, settings, serviceUrl) {
     }
   };
 
+  this.orgName = function (groupId) {
+    const group = this.groups.get(groupId);
+    return group && group.organization && group.organization.name;
+  };
+
+  this.sortedGroups = function () {
+    if (this.groups.all()) {
```

Comment:
This will always be truthy, since `groups.all()` always returns an array, never `null` or `undefined`.